indico/modules/events/abstracts/controllers/management.py

```diff
@@ -87,7 +88,9 @@ class RHAbstracts(RHManageAbstractsBase):
     """Display abstracts management page"""
 
     def _process(self):
-        return WPManageAbstracts.render_template('management/abstracts.html', self._conf, event=self.event_new)
+        abstracts_count = Abstract.query.with_parent(self.event_new).with_entities(db.func.count()).scalar()
```

Comment:
why so complicated and not just `Abstract.query.with_parent(self.event_new).count()`? in registration we needed the more complex query since you had multiple regforms